src/Microsoft.Extensions.WebSockets.Internal/Microsoft.Extensions.WebSockets.Internal.csproj

```diff
@@ -15,7 +15,8 @@
   <ItemGroup>
     <PackageReference Include="System.IO.Pipelines" Version="$(CoreFxLabsVersion)" />
     <PackageReference Include="System.IO.Pipelines.Text.Primitives" Version="$(CoreFxLabsVersion)" />
-    <PackageReference Include="Microsoft.Extensions.TaskCache.Sources" Version="1.2.0-*" PrivateAssets="All"/>
+    <PackageReference Include="Microsoft.Extensions.TaskCache.Sources" Version="1.2.0-*" PrivateAssets="All" />
+    <PackageReference Include="System.ValueTuple" Version="4.3.0" />
```

Comment:
Nope, we want 4.3.0 explicitly here. I originally had it as $(CoreFxVersion) and things broke. I'll double check why and make sure bugs are filed if we still need it